support-workers/src/main/scala/com/gu/support/workers/lambdas/CreatePaymentMethod.scala

```diff
@@ -154,7 +150,18 @@ class CreatePaymentMethod(servicesProvider: ServiceProvider = ServiceProvider)
       .retrieveEmail(payPal.baid)
       .map(PayPalReferenceTransaction(payPal.baid, _))
 
-  def createDirectDebitPaymentMethod(dd: DirectDebitPaymentFields, user: User): Future[DirectDebitPaymentMethod] = {
+  def createDirectDebitPaymentMethod(
+      dd: DirectDebitPaymentFields,
+      state: CreatePaymentMethodState,
+  ): Future[DirectDebitPaymentMethod] = {
+    import state.user
+
+    // We only want to send the email address to the Tortoise Media gateway, the GNM gateway doesn't need it
+    val (email, paymentGateway) = state.product match {
+      case paper: Paper if paper.productOptions == Sunday =>
+        (Some(state.user.primaryEmailAddress), DirectDebitTortoiseMediaGateway)
```

Comment:
minor but since you're importing state.user you can just refer to this as user
```suggestion
        (Some(user.primaryEmailAddress), DirectDebitTortoiseMediaGateway)
```

Reply:
@andresilva-guardian is this email address something we also need to consider in MDAPI or will it get migrated automatically?

Reply:
This code is no longer needed as the Tortoise account has been reconfigured.